buildmetrics/buildmetrics.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux || darwin || windows
```

Comment:
what side effect do this have?

Reply:
It's that it only builds for those targets
For other targets there is a corresponding side implementation

Almost like an hardware/target interface

In this case there is a _JS file that nil implements JS bits

Reply:
Cool, thanks!